src/test/java/de/retest/web/DefaultValuesProviderTest.java

```diff
@@ -0,0 +1,15 @@
+package de.retest.web;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+class DefaultValuesProviderTest {
+
+	@Test
+	void can_read_file() {
+		assertThat( new DefaultValuesProvider().isDefault( "blockquote", "display", "block" ) ).isTrue();
```

Comment:
`DefaultValuesProvider cut = new DefaultValuesProvider();` Since you load the values every time... .

Reply:
Good catch!